data/src/server.rs

```diff
@@ -77,62 +103,62 @@ impl Map {
         for (_, config) in self.0.iter_mut() {
             if let Some(pass_file) = &config.password_file {
                 if config.password.is_some() || config.password_command.is_some() {
-                    return Err(Error::Parse(
-                        "Only one of password, password_file and password_command can be set."
-                            .to_string(),
-                    ));
+                    return Err(Error::Parse(DUP_PASS_MSG.to_string()));
```

Comment:
Nit: Just make these separate Error variants so we don't need to use `Parse` plus some const string for each error type.